### DISCUSSION TITLE
example for new cask with only a *.jar file

### DISCUSSION THREAD

**Discussion by GETandSELECT** [Homebrew/discussions#27] [Everyday usage] [answered]
Hi

I wish to create a cask for https://github.com/RipMeApp/ripme

It's only a *.jar

Do you know a similar app / example?

**Comment by SMillerDev** [comment]
Why not just make it a formula?

**Comment by tomtastic** [comment, marked as answer]
Maybe look at the Recaf cask, it avoids the horrible shim and just leverages macOS' Jar Launcher.
